docs/source/api/apollo-server.mdx

```diff
@@ -15,18 +13,19 @@ Takes an `options` object as a parameter. Supported fields of this object are de
 
 ### Example
 
-```js
+<MultiCodeBlock>
+
+```ts
+import { ApolloServer } from '@apollo/server';
+
 const server = new ApolloServer({
 	typeDefs,
 	resolvers,
-  csrfPrevention: true,  // highly recommended
-  cache: 'bounded',
-  plugins: [
-    ApolloServerPluginLandingPageLocalDefault({ embed: true }),
-  ],
 });
 ```
 
+</MultiCodeBlock>
+
 ### Options
```

Comment:
I really do wish we didn't end up with side-scrolling even on reasonably-sized displays. Maybe there's another approach we can take to this information that isn't a two column table? Something a bit more `<dt><dd>`ish?

Reply:
I think I'm going to add this to my TODO list post-merging this PR if that sounds okay to you. I'll experiment and find a better way to display this info, but I'd rather merge this PR first then come back and iterate later.